www/_data/redirects.yml

```diff
@@ -13,6 +13,8 @@ docs:
     - {old: "dev/guide/platforms/ios/config.html", new: "dev/config_ref/index.html"}
     - {old: "latest/guide/platforms/ios/tools.html", new: "latest/guide/platforms/ios/index.html"}
     - {old: "latest/guide/platforms/ios/config.html", new: "latest/config_ref/index.html"}
+    - {old: "dev/guide/platforms/win8/packaging.html", new: "dev/guide/platforms/win8/index.html#signing-an-app"}
+    - {old: "dev/guide/platforms/win8/win10-support.html", new: "dev/guide/platforms/win8/index.html"}
```

Comment:
Okay. The packaging one has been live since long so I will keep that one. Win10 support didn't went out so I will change it to latest.